.clang-tidy

```diff
@@ -26,9 +26,18 @@ Checks:     '
             performance-*,
             portability-*,
             readability-*,
+            -bugprone-exception-escape,
+            -bugprone-macro-parentheses,
+            -bugprone-too-small-loop-variable,
+            -clang-analyzer-cplusplus.NewDelete,
+            -clang-analyzer-cplusplus.NewDeleteLeaks,
             -google-readability-braces-around-statements,
-            -google-readability-namespace-comments,
+            -modernize-use-nodiscard,
+            -modernize-avoid-c-arrays,
+            -performance-unnecessary-value-param,
             -readability-braces-around-statements,
+            -readability-isolate-declaration,
+            -readability-magic-numbers,
```

Comment:
Yells at us about magic numbers mostly in tests. Not gonna worry about this check.

Reply:
We could disable it just for tests, I think, if that is an issue. I do remember someone going ??? at magic numbers before.